src/js/view/controls/settings-menu.js

```diff
@@ -179,7 +180,12 @@ export function setupSubmenuListeners(settingsMenu, controlbar, model, api) {
             const mediaModel = model.get('mediaModel');
             onAudiotracksChanged(mediaModel, mediaModel, mediaModel.get('audioTracks'));
             onQualitiesChanged(model, mediaModel.get('levels'));
-            onPlaybackRatesChange(model, model.get('playbackRates'));
+            setupPlaybackRatesMenu(model, model.get('playbackRates'));
         }
     });
+
+    model.on('change:streamType', (changedModel, newStreamType) => {  
```

Comment:
Not using the `newStreamType` argument caused the build to fail:
```
...src/js/view/controls/settings-menu.js
  187:50  error  'newStreamType' is defined but never used  no-unused-vars
```
Run grunt lint:player to confirm your changes meet lint rules.